src/transformStateWithClones.js

```diff
@@ -7,7 +7,49 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const clone = { ...state };
```

Comment:
Be more specific with names
```suggestion
  const stateClone = { ...state };
```